### ISSUE TITLE
Irrelevant mpris notifications

### ISSUE THREAD

**Issue by sonnyp** [sonnyp/Tangram#207]
![Screenshot from 2023-01-04 21-50-27](https://user-images.githubusercontent.com/19673/210651808-3ab280ae-2794-4e59-9713-65c931b3a4e1.png)

**Comment by philn** [comment]
https://bugs.webkit.org/show_bug.cgi?id=247527

**Comment by sonnyp** [comment]
Do you know if there is a way to disable mpris at runtime? I couldn't find anything.

**Comment by philn** [comment]
Not yet, but we should make it possible actually before 2.40:

https://github.com/WebKit/WebKit/blob/main/Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp#L202

**Comment by ivan-avalos** [comment]
Maybe it is a bug in WebKit? MPRIS notifications should only show up when media is playing. You shouldn't have to manually enable it or disable it.

**Comment by philn** [comment]
Yes, it's a WebKit bug.

**Comment by xgqt** [comment]
Looks like this is gone with webkit 2.38.5 (2.38.5-r500 on Gentoo).

**Comment by philn** [comment]
yes iirc we disabled the feature in WebKitGTK until someone investigates the issue...
